### ISSUE TITLE
Spacing around generics

### ISSUE THREAD

**Issue by swankjesse** [pinterest/ktlint#580]
It doesn’t complain where I would prefer it to.

```
  var a: Map< Int, String> = mapOf()
  var b: Map<Int, String > = mapOf()
  var c: Map <Int, String> = mapOf()
```

**Comment by romtsn** [comment]
Fixed by #769 (put into experimental ruleset for now)
